src/app/app.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+import { Title } from "@angular/platform-browser";
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss'],
+})
+export class AppComponent {
+  private titleService: Title;
+
+  public loggedUser: string = 'User1';
```

Comment:
Better to make separate service that is related to authentication purposes. Something like `AuthService`.